### PR TITLE
StreamerPlugin: changes syncup from buildroot main

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: recipes-multimedia/aamp/aamp_git.bb [hunk 1]
```diff
@@ -15,21 +15,27 @@ PV = "0.1.gitr${SRCPV}"
 SRC_URI = "\
```
Comment (404ag) [inline recipes-multimedia/aamp/aamp_git.bb:15]: Can you please add BRANCH variable and make a weak assignment for SRCREV and BRANCH?

File: recipes-multimedia/aamp/aampabr_git.bb [hunk 2]
```diff
@@ -8,7 +8,7 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=175792518e4ac015ab6696d16c4f607e"
 PV = "0.1.gitr${SRCPV}"
 
 SRC_URI = "git://github.com/rdkcmf/rdk-aampabr.git;branch=stable2;protocol=https"
```
Comment (404ag) [inline recipes-multimedia/aamp/aampabr_git.bb:10]: Can you please add BRANCH variable and make a weak assignment for SRCREV and BRANCH?

File: recipes-multimedia/aamp/aampmetrics_git.bb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+DESCRIPTION = "Metrics Library is an independent library for session statistics generation/reporting in JSON format"
+HOMEPAGE = "https://github.com/rdkcmf/rdk-aampmetrics"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=175792518e4ac015ab6696d16c4f607e"
+
+DEPENDS_append = " cjson"
+
+PV = "0.1.gitr${SRCPV}"
+
+SRC_URI = "git://github.com/rdkcmf/rdk-aampmetrics.git;branch=stable2;protocol=https"
```
Comment (404ag) [inline recipes-multimedia/aamp/aampmetrics_git.bb:11]: Can you please add BRANCH variable and make a weak assignment for SRCREV and BRANCH?

File: recipes-multimedia/aamp/aampmetrics_git.bb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+SRCREV = "c6b9de63f2a8755db59ee300c02c6c89e392dad5"
+
+S = "${WORKDIR}/git"
+
+inherit cmake
+
+PACKAGES = "${PN} ${PN}-dev ${PN}-dbg"
```
Comment (404ag) [inline recipes-multimedia/aamp/aampmetrics_git.bb:18]: Is this needed? I think these packages should be enabled by default. No need to specify them explicitly

File: recipes-multimedia/aamp/aamp_git.bb [hunk 5]
```diff
@@ -15,21 +15,27 @@ PV = "0.1.gitr${SRCPV}"
...
     git://github.com/rdkcmf/rdk-aamp.git;protocol=https;branch=stable2 \
     file://0001-rdk-aamp-disable-getsourceid-chech-temporarily-to-sendsyncevent.patch \
+    file://0002-rdk-amp-align-ocdm-drm-adapter-interface.patch \
 "
-SRCREV = "24a4dfa609c7c6b3c18ee595bde0874c9a5ca7c9"
```
Comment (404ag) [inline recipes-multimedia/aamp/aamp_git.bb:19]: Can you please add BRANCH variable and make weak assignment for BRANCH and SRCREV?

File: recipes-multimedia/aamp/gst-plugins-rdk-aamp_git.bb [hunk 6]
```diff
@@ -8,8 +8,11 @@ DEPENDS_append = " aamp gstreamer1.0 gstreamer1.0-plugins-base wpeframework-clie
 
 PV = "0.1.gitr${SRCPV}"
 
-SRC_URI = "git://github.com/rdkcmf/rdk-gst-plugins-rdk-aamp.git;protocol=https;branch=stable2"
-SRCREV = "c9d03d4df04e0a09a0b8cd45c122c6add12b0501"
+SRC_URI = "\
```
Comment (404ag) [inline recipes-multimedia/aamp/gst-plugins-rdk-aamp_git.bb:11]: Can you please add BRANCH variable and make weak assignment for BRANCH and SRCREV?

File: recipes-multimedia/cenc/gstreamer1.0-plugins-cencdecrypt_git.bb [hunk 7]
```diff
@@ -16,7 +17,7 @@ DEPENDS = " \
 PV = "0.1.gitr${SRCPV}"
 
 SRC_URI = "git://github.com/WebPlatformForEmbedded/gstcencdecryptor.git;protocol=https;branch=master"
```
Comment (404ag) [inline recipes-multimedia/cenc/gstreamer1.0-plugins-cencdecrypt_git.bb:19]: Can you please add BRANCH variable and make weak assignment for BRANCH and SRCREV?

**Review by 404ag — CHANGES_REQUESTED** [review]

File: recipes-multimedia/aamp/aampmetrics_git.bb (hunk 4 above)
Comment (HaseenaSainul) [reply recipes-multimedia/aamp/aampmetrics_git.bb:18]: It seems some more additional things are getting added and if I remove this it force be to add INSANE_SKIP_${PN}-dev = "dev-elf" and finally wpeframework-plugins sendup with wpeframework-plugins rdepends on aamp-dev [dev-deps]

Comment (404ag) [reply recipes-multimedia/aamp/aampmetrics_git.bb:18]: ok, then please leave it like it is

File: recipes-multimedia/aamp/aamp_git.bb (hunk 1 above)
Comment (HaseenaSainul) [reply recipes-multimedia/aamp/aamp_git.bb:16]: done

File: recipes-multimedia/aamp/aampmetrics_git.bb (hunk 4 above)
Comment (HaseenaSainul) [reply recipes-multimedia/aamp/aampmetrics_git.bb:19]: done

File: recipes-multimedia/aamp/gst-plugins-rdk-aamp_git.bb (hunk 6 above)
Comment (HaseenaSainul) [reply recipes-multimedia/aamp/gst-plugins-rdk-aamp_git.bb:12]: done

File: recipes-multimedia/cenc/gstreamer1.0-plugins-cencdecrypt_git.bb (hunk 7 above)
Comment (HaseenaSainul) [reply recipes-multimedia/cenc/gstreamer1.0-plugins-cencdecrypt_git.bb]: done

File: recipes-multimedia/aamp/aamp_git.bb (hunk 5 above)
Comment (HaseenaSainul) [reply recipes-multimedia/aamp/aamp_git.bb:19]: done

**Review by 404ag — APPROVED** [review]

**Pull request merged**